thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -254,6 +260,8 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
       sessionId,
       sessionName,
       sessionText,
+      sessionOwner,
```

Comment:
maybe rename this to "sessionUpdatedBy". The session-owner is the user that originally saved it, which may or may not be the last person to update it.

Reply:
Gotcha. That makes sense. Will make the change.